test/controllers/jobSeeker.test.ts

```diff
@@ -4,51 +4,171 @@ import request from 'supertest'
 import connection from '../db-helper'
 import app from '../../src/app'
 
+process.on('uncaughtException', function (err) {
+  console.log('oh no', err);
+});
+
 jest.mock(
   '../../src/middlewares/tokenVerify',
   () => (req: Request, res: Response, next: NextFunction) => next()
 )
-const form = {
+
+const creatingSkills = async () => {
+  await request(app).post('/skills/create').send({ name: 'javascript' })
+  await request(app).post('/skills/create').send({ name: 'reactjs' })
+  await request(app).post('/skills/create').send({ name: 'typescript' })
+  await request(app).post('/skills/create').send({ name: 'react' })
+  await request(app).post('/skills/create').send({ name: 'nodejs' })
+  await request(app).post('/skills/create').send({ name: 'docker' })
+  await request(app).post('/skills/create').send({ name: 'aws' })
+  await request(app).post('/skills/create').send({ name: 'gcp' })
+  await request(app).post('/skills/create').send({ name: 'iac' })
+}
+
+const jobPost1Skills = async () => {
+  await request(app).get('/skills/1').send({ name: 'javascript' })
+  await request(app).get('/skills/2').send({ name: 'reactjs' })
+  await request(app).get('/skills/5').send({ name: 'nodejs' })
```

Comment:
why do you need to use `send`, this is just for querying the skills

Reply:
Oh yeah, it doesn't need to send the skill to the screen. :))